tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+4. Multiple tool calls
+5. End2End tool calling
+6. Automatic function calling
+7. Image (url)
+8. Image (base64)
+9. Multiple images
+10. Complete end2end test with conversation history, tool calls, tool results and images
+11. Integration specific tests
+"""
+
+import pytest
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)